src/ukify/ukify.py

```diff
@@ -1371,6 +1441,15 @@ def config_example(self) -> tuple[Optional[str], Optional[str], Optional[str]]:
         config_key = 'UKI/Profile',
     ),
 
+    ConfigItem(
+        '--add-profile',
+        dest = 'add_profiles',
```

Comment:
you mapped an option named in singular form into a variable in plural. is that intended or an accident?

Reply:
Not an accident, you specify it multiple times and the results are aggregated into a single list so the variable name should be plural